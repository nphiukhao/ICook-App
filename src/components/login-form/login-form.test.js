import React from 'react'
import LoginForm from './login-form'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`login form component`, () => {
    it('Login form snapshot test', () => {
        const tree = shallow(<LoginForm/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
})