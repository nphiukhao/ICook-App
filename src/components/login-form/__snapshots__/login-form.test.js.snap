// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login form component Login form snapshot test 1`] = `
<div
  className="login-container"
>
  <p>
    login
  </p>
  <form
    onSubmit={[Function]}
  >
    <input
      id="user_name"
      name="user_name"
      placeholder="Username"
      required={true}
    />
    <input
      id="password"
      name="password"
      placeholder="Password"
      required={true}
      type="password"
    />
    <button
      type="submit"
    >
      login
    </button>
  </form>
</div>
`;
